attribute/value.go

```diff
@@ -242,6 +267,8 @@ func (v Value) Emit() string {
 		return fmt.Sprint(v.asStringSlice())
 	case STRING:
 		return v.stringly
+	case BYTESLICE:
+		return fmt.Sprintf("%q", v.asByteSlice())
```

Comment:
I'm not sure emitting as a quoted string is the most appropriate here. It assumes the slice contains some encoded string which is not guaranteed to be true. Why not just emit `fmt.Sprint(v.asByteSlice())`?

Reply:
My thought process here was to escape values so it would escape whitespace characters. If that's not the case, I'm happy to change it to `fmt.Sprint(v.asByteSlice())

Reply:
The worry I have is this approach assumes the byte slice will be used only for unicode encoded data. Many other user byte slices, like trace and span IDs, should not be displayed as text though.

Reply:
Sounds good, I updated it to your recommendation.